pkg/resources/execute_acceptance_test.go

```diff
@@ -704,20 +707,68 @@ func TestAcc_UnsafeExecute_queryResultsBug(t *testing.T) {
 	})
 }
 
-func unsafeExecuteConfig(queryNumber int) string {
+func executeConfig(queryNumber int) string {
 	return fmt.Sprintf(`
-resource "snowflake_unsafe_execute" "test" {
+resource "snowflake_execute" "test" {
   execute = "SELECT 18"
   revert  = "SELECT 36"
   query  = "SELECT %d"
 }
 
-output "unsafe" {
-  value = snowflake_unsafe_execute.test.query_results
+output "query_results_output" {
+  value = snowflake_execute.test.query_results
 }
 `, queryNumber)
 }
 
+func TestAcc_Execute_QueryResultsRecomputedWithoutQueryChanges(t *testing.T) {
```

Comment:
hmmm, is it good though? in the docs we have: `Optional SQL statement to do a read. Invoked after creation and every time it is changed.` and here it is run even without being changed - so we have to either:
- change the docs
- change the implementation (e.g. completely empty read and filling it only in create/update?)

Let's talk

Reply:
As discussed adjusted the docs